.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: go
 
 install:
+  - A=${PWD#*github.com/};A=${A%/ffjson};cd ../..;mv $A pquerna;cd pquerna/ffjson
```

Comment:
When building in a forked repo on public Travis, go install will install the original package which is pquerna/ffjson. This line moves whatever xxxx/ffjson to pquerna/ffjson before install/build.

I think it's good to have it to make it easier for other committers to test on Travis first before PR.